Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -2227,6 +2228,9 @@ def prepare(self, doChecking = False):
             self.pythonCfgCode +="process.options.numberOfThreads=cms.untracked.uint32("+self._options.nThreads+")\n"
             self.pythonCfgCode +="process.options.numberOfStreams=cms.untracked.uint32("+self._options.nStreams+")\n"
             self.pythonCfgCode +="process.options.numberOfConcurrentLuminosityBlocks=cms.untracked.uint32("+self._options.nConcurrentLumis+")\n"
+            self.process.options.numberOfThreads=cms.untracked.uint32(int(self._options.nThreads))
```

Comment:
these lines are an unrelated fix that is not needed for igprof in my understanding, but does not harm either